pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -128,13 +133,18 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-.PHONY: docker-build
-docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+.PHONY: container-build
+container-build: test ## Build container image with the manager.
```

Comment:
We cannot address breaking changes in the plugins.
To do that we would need to for e.g.:
- Create another plugin
- Update the docs 


So, what about we have the options by vars which helps users customize and change as they wish but does not change the names of the target? I would be ok with that.

Reply:
I can live with that. But could you ping me when a new plugin is planning on being released so I can add the new targets?

Reply:
you can create an issue and we can link to : https://github.com/kubernetes-sigs/kubebuilder/milestone/26
for now, we do have not enough requirements that would justify that.

Reply:
as this is stale for a while, I might need a quick session on how you envision this and I'll do as you requested :)

and maybe I can answer my stuff with a question:
1. would you want me to break the change to be without the new targets for now, and change them in the new release
2. create an issue and add the targets in the new plugin

Reply:
/hold 

I do not think that we should change the name of these targets now
It can be very disruptive and that would mean changing all docs and etc as well.

Reply:
Also, with this PR would came all changes in the docs

Reply:
I will create a new comment